package/debian/summary

```diff
@@ -0,0 +1,10 @@
+ FreeCAD is an Open Source CAx RAD based on OpenCasCade, Qt and Python.
+ It features some key concepts like macro recording, workbenches, ability
+ to run as a server and dynamically loadable application extensions and
+ it is designed to be platform independent.
+ .
+ Currently, FreeCAD can import and display CAD models in IGES, STEP, and
+ BRep formats and meshes in STL, BMS, AST and Wavefront OBJ formats.
+ Editing and modeling features are currently somewhat limited.
```

Comment:
Where did you get this text from? This is clearly outdated information. I remember there was updated information in some other package.

Reply:
@vocx-fc https://packages.debian.org/sid/freecad

Reply:
Updated description.
This is a blend of deb and wiki descriptions.